lib/authie/session_model.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'secure_random_string'
+require 'authie/config'
+
+module Authie
+  class SessionModel < ActiveRecord::Base
```

Comment:
I'd suggest requiring `active_record/base` in this file to ensure the `SessionModel` class can be parsed if authie is loaded before/without ActiveRecord/Rails.

Reply:
Done